src/main/resources/init_db.sql

```diff
@@ -1,9 +1,17 @@
-CREATE SCHEMA IF NOT EXISTS `taxi` DEFAULT CHARACTER SET utf8;
-USE `taxi`;
+CREATE DATABASE `taxi_service` /*!40100 DEFAULT CHARACTER SET utf8 */ /*!80016 DEFAULT ENCRYPTION='N' */;
```

Comment:
why the second part is commented out?

Reply:
that is a copy from mySQL

Reply:
I understand, but this shouldn't be commented out